deps/obs-scripting/obs-scripting-python.c

```diff
@@ -1654,7 +1654,7 @@ bool obs_scripting_load_python(const char *python_path)
 	/* ---------------------------------------------- */
 	/* Load main interface module                     */
 
-	add_to_python_path(SCRIPT_DIR);
+	add_to_python_path(os_get_abs_path_ptr(SCRIPT_DIR));
```

Comment:
This will cause a memory leak. The returned value from `os_get_abs_path_ptr()` is expected to be freed by the caller.

Reply:
Just force-pushed an update to this commit to fix that.